test/index.js

```diff
@@ -2,197 +2,196 @@ const fs = require('fs');
 const path = require('path');
 const { expect } = require('chai');
 let hb;
+let blob, face, font, buffer;
 
 before(async function () {
   hb = await require('..');
 });
 
 afterEach(function () {
-  if (this.blob) this.blob.destroy();
-  if (this.face) this.face.destroy();
-  if (this.font) this.font.destroy();
-  if (this.buffer) this.buffer.destroy();
-  this.blob = this.face = this.font = this.buffer = null;
+  if (blob) blob.destroy();
+  if (face) face.destroy();
+  if (font) font.destroy();
+  if (buffer) buffer.destroy();
+  blob = face = font = buffer = undefined;
```

Comment:
Oh whoops. Apparently shared context via `this` only works if the `afterEach` is defined in a `describe`. I think I like this fix since it's compatible with the new-ish `node:test`.